logger/logger.go

```diff
@@ -24,6 +24,12 @@ const DefaultPrefix = "instana: "
 // Level defines the minimum logging level for logger.Log
 type Level uint8
 
+// Less returns whether the log level is less than the given one in logical order:
+// ErrorLevel > WarnLevel > InfoLevel > DebugLevel
+func (lvl Level) Less(other Level) bool {
+	return uint8(lvl) > uint8(other)
```

Comment:
I think we do not require type cast here. What do you think?
```suggestion
	return lvl > other
```

Reply:
I agree that it's redundant from the code perspective, however I think it increases readability explicitly telling that we compare integers there.